epubmaker_postprocessing.rb

```diff
@@ -241,24 +275,26 @@ def sendAlertMail(epubcheck_output, testing_value_file, message, logkey='')
 
 # ---------------------- PROCESSES
 
-data_hash = readConfigJson('read_config_json')
+# If an epubcheck_errfile exists, delete it
+deleteFileIfPresent(epubcheck_errfile, 'delete_epubcheck_errfile')
+
+data_hash = readJson(Metadata.configfile, 'read_config_json')
 #local definition(s) based on config.json
 stage_dir = data_hash['stage']
 
-# If an epubcheck_errfile exists, delete it
-deleteFileIfPresent(epubcheck_errfile, 'delete_epubcheck_errfile')
+styleconfig_hash = readJson(styleconfig_json, 'read_styleconfig_json')
 
-# Add links back to TOC to chapter heads
-addLinkstoTOC(oebps_dir, "ch[0-9][0-9]*.html", epubmakerpostprocessingjs, 'add_links_to_TOC_to_chap_heads')
+htmlfilenames = getHTMLfilenameTypes(styleconfig_hash, 'get_html_filename-types')
 
-# Add links back to TOC to appendix heads
-addLinkstoTOC(oebps_dir, "app[0-9][0-9]*.html", epubmakerpostprocessingjs, 'add_links_to_TOC_to_appendix_heads')
+htmlfileshortnames = getHTMLfileShortNames(chunk_xsl, 'get_html_file_short_names')
 
-# Add links back to TOC to preface heads
-addLinkstoTOC(oebps_dir, "preface[0-9][0-9]*.html", epubmakerpostprocessingjs, 'add_links_to_TOC_to_preface_heads')
+# combine the 2 arrays of html filename prefix-possiblities
+htmlfilenames.concat htmlfileshortnames
```

Comment:
Ah ok cool, so the XSL only abbreviates *some* filenames, but others it leaves intact. Gotcha!